link_resolver_track_list.go

```diff
@@ -0,0 +1,205 @@
+package main
+
+import (
+	"bytes"
+	"encoding/json"
+	"errors"
+	"fmt"
+	"io/ioutil"
+	"log"
+	"net/http"
+	"net/url"
+	"regexp"
+	"strconv"
+	"strings"
+	"text/template"
+	"time"
+)
+
+var (
+	invalidTrackPath = errors.New("invalid track list track path")
+)
+
+func init() {
+	const (
+		trackListAPIURL = "https://supinic.com/api/track/detail/%d"
+
+		tooltipTemplate = `<div style="text-align: left;">
+<b>{{.Name}}</b><br>
+<b>Music Track</b><br>
```

Comment:
Is this necessary? It looks a bit useless to me. Can be replaced with an empty line to separate title from details.

Reply:
Changed in `e60a123`